.changes/next-release/bugfix-2644a585-8c32-44ad-8bda-eccc34100ca4.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
The changelog is more for users of the toolkit than a general changelog. I have tried to clarify this in #2847

For that reason I think we can remove this entry.